src/main/groovy/org/transmartproject/batch/highdim/acgh/data/AcghDataStepsConfig.groovy

```diff
@@ -92,7 +94,10 @@ class AcghDataStepsConfig implements StepBuildingConfigurationTrait {
         TaskletStep step = steps.get('secondPass')
                 .chunk(dataFilePassChunkSize)
                 .reader(acghDataTsvFileReader())
-                .processor(patientInjectionProcessor())
+                .processor(compositeOf(
+                    filterDataWithoutAssayMappingsItemProcessor(),
+                    patientInjectionProcessor()
+                ))
```

Comment:
I think it would be more clear and efficient to have the filter actually always working (instead of disabling itself depending of a job parameter) and decide whether to use the filter or not during step creation.